src/client.cpp

```diff
@@ -295,21 +284,31 @@ ADDON_STATUS ADDON_SetSetting(const char *settingName, const void *settingValue)
   }
   else if (str == "livestreamingmethod")
   {
-    eStreamingMethod  setting_livestreamingmethod = *(eStreamingMethod*) settingValue;
-    if (g_livestreamingmethod == ClientTimeshift)
+    if (g_client)
     {
-        if (setting_livestreamingmethod == RealTime)
+      if (g_client->GetBackendVersion() < 50000)
+      {
+        eStreamingMethod  setting_livestreamingmethod = *(eStreamingMethod*)settingValue;
```

Comment:
One space and not two and C++ style cast

Reply:
Old code